tudatBundle/pagmo2/include/pagmo/algorithms/giACOmo.hpp

```diff
@@ -79,73 +79,73 @@ class giACOmo
     */
-    giACOmo(unsigned gen = 1u, double acc = 0.95, unsigned int FSTOP = 1, unsigned int IMPSTOP = 1, unsigned int EVALSTOP = 1,
-          double FOCUS = 0.9,  unsigned int KER = 10, double ORACLE=1.0, unsigned int PARETOMAX = 10,
-            double EPSILON = 0.9, unsigned seed = pagmo::random_device::next())
-        : m_gen(gen), m_acc(acc), m_FSTOP(FSTOP), m_IMPSTOP(IMPSTOP), m_EVALSTOP(EVALSTOP), m_FOCUS(FOCUS),
-          m_KER(KER), m_ORACLE(ORACLE), m_PARETOMAX(PARETOMAX), m_EPSILON(EPSILON), m_e(seed), m_seed(seed), m_verbosity(0u),
+    gi_aco_mo(unsigned gen = 1u, double acc = 0.95, unsigned int fstop = 1, unsigned int impstop = 1, unsigned int evalstop = 1,
+          double focus = 0.9,  unsigned int ker = 10, double oracle=1.0, unsigned int paretomax = 10,
+            double epsilon = 0.9, unsigned seed = pagmo::random_device::next())
+        : m_gen(gen), m_acc(acc), m_fstop(fstop), m_impstop(impstop), m_evalstop(evalstop), m_focus(focus),
+          m_ker(ker), m_oracle(oracle), m_paretomax(paretomax), m_epsilon(epsilon), m_e(seed), m_seed(seed), m_verbosity(0u),
           m_log()
     {
         if (acc >= 1. || acc < 0.) {
             pagmo_throw(std::invalid_argument, "The accuracy parameter must be in the [0,1[ range, while a value of "
                                                    + std::to_string(acc) + " was detected");
         }
-        if (FSTOP < 0.) {
+        if (fstop < 0.) {
             pagmo_throw(std::invalid_argument, "The objective stopping criterion must be in the [0,inf[ range, while a value of "
-                                                   + std::to_string(FSTOP) + " was detected");
+                                                   + std::to_string(fstop) + " was detected");
         }
-        if (IMPSTOP < 0) {
+        if (impstop < 0) {
             pagmo_throw(std::invalid_argument,
                         "The improvement stopping criterion must be in [0, inf[, while a value of "
-                            + std::to_string(IMPSTOP) + " was detected");
+                            + std::to_string(impstop) + " was detected");
         }
-        if (EVALSTOP < 0) {
+        if (evalstop < 0) {
             pagmo_throw(std::invalid_argument,
                         "The evaluation stopping criterion must be in [0, inf[, while a value of "
-                            + std::to_string(EVALSTOP) + " was detected");
+                            + std::to_string(evalstop) + " was detected");
         }
 
-        if (FOCUS >= 1. || FOCUS < 0.) {
+        if (focus >= 1. || focus < 0.) {
             pagmo_throw(std::invalid_argument,
                         "The focus parameter must be in [0, inf[, while a value of "
-                            + std::to_string(FOCUS) + " was detected");
+                            + std::to_string(focus) + " was detected");
         }
 
-        if (KER < 0) {
+        if (ker < 0) {
```

Comment:
see above, and checkl all of the checks on the parameters. Will not comment on those anymore.

1) if unsigned the bound   >=0 is automatically respected
2) do not use inf as a bound. Thats ok in math in computer science use > < etc...